debian/control

```diff
@@ -34,6 +34,8 @@ Depends: ${shlibs:Depends},
          sssd,
          sssd-dbus,
          apparmor,
+         cifs-utils,
+         curlftpfs,
```

Comment:
This is in universe, so we either downgrade it to suggests + add some documentation in the system mount adsys doc page or file a MIR for it.

I think suggestion number 1.